### PR TITLE
Add mongodb driver

### PR DESCRIPTION
**Pull request by wgordon17**
Signed-off-by: Will Gordon <wgordon@redhat.com>

### CODE REVIEW

File: 7.0/Dockerfile [hunk 1]
```diff
@@ -55,6 +55,8 @@ RUN sed -i -f /opt/app-root/etc/httpdconf.sed /opt/rh/httpd24/root/etc/httpd/con
     echo "IncludeOptional /opt/app-root/etc/conf.d/*.conf" >> /opt/rh/httpd24/root/etc/httpd/conf/httpd.conf && \
     mkdir /tmp/sessions && \
     chown -R 1001:0 /opt/app-root /tmp/sessions && \
+    #chown -R 1001:0 `pecl config-get php_dir`/.. && \
+    #chmod -R ug+rwx `pecl config-get php_dir`/.. && \
```
Comment (omron93) [inline 7.0/Dockerfile]: Why is this needed?

**Comment by omron93** [comment]
Thank you for this PR.

@hhorak What about installing community SCL packages? What about a support for this?

@notroj @remicollet What do you think?

[test]

**Comment by remicollet** [comment]
> What about installing community SCL packages? What about a support for this?

This will make image diverge from RH official image.

File: 7.0/Dockerfile (hunk 1 above)
Comment (wgordon17) [reply 7.0/Dockerfile]: Apologies, it's not needed at all

**Comment by wgordon17** [comment]
@remicollet Does updating the Dockerfile.rhel7 files solve your reservation?

**Comment by remicollet** [comment]
@wgordon17 no, as the mongodb extension is not available in RHSCL.

**Comment by hhorak** [comment]
I don't think we should diverge between centos and rhel images, moving from community images (centos) to supported images (rhel) should not require any changes. I'd rather think about whether there is another way how to install the the extension. In case of ruby for example, all such missing connectors are installed as any other missing dependency, from upstream repository. Is something like that possible for php?

**Comment by remicollet** [comment]
For pur-PHP library, can be installed using "composer" (already managed by the image)
For C extension, can be install using "pecl", btw, this requires a C compiler, and devel packages

     pecl install mongodb-1.3.0

**Comment by hhorak** [comment]
@wgordon17, would installing by composer be acceptable solution for you?

**Comment by remicollet** [comment]
@hhorak mongodb is a C extension (so s/composer/pecl/)

**Comment by wgordon17** [comment]
@remicollet @hhorak `pecl install mongodb` was my first thought, and in trying to figure out permissions, I just assumed a `yum install` would've been easier. But I can certainly revisit the permissions needed for `pecl install` to work.

**Comment by omron93** [comment]
@wgordon17 Any progress in this?

It seems that  @maicolcantagallo is facing same problem in https://github.com/sclorg/s2i-php-container/issues/174

**Comment by wgordon17** [comment]
I have not had any progress yet. Apologies for the delay.


From: centos-ci <notifications@github.com> <notifications@github.com>
Reply: sclorg/s2i-php-container
<reply@reply.github.com>
<reply@reply.github.com>
Date: November 28, 2017 at 10:59:18
To: sclorg/s2i-php-container <s2i-php-container@noreply.github.com>
<s2i-php-container@noreply.github.com>
CC: Will Gordon <wgordon@redhat.com> <wgordon@redhat.com>, Mention
<mention@noreply.github.com> <mention@noreply.github.com>
Subject:  Re: [sclorg/s2i-php-container] Add mongodb driver (#169)

Can one of the admins verify this patch?

**Comment by phracek** [comment]
This ticket can be closed. MongoDB is not supported at all. Feel free to reopen it.
